python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -159,4 +170,122 @@ basis : list of :class:`~openturns.Function`
+    a_{\vect{\alpha}} \psi_{\vect{\alpha}}(\standardInputObservation)
+
+for any :math:`\standardInputObservation \in \standardInputSpace`. 
+Assume that the input marginals :math:`\{Z_i\}_{i = 1, ..., \physicalInputDimension}` are independent. 
+Let :math:`\vect{u} \subseteq \{1, ..., \physicalInputDimension\}` be a group 
+of variables with dimension :math:`\operatorname{card}(\vect{u}) \in \mathbb{N}`. 
+Assume that :math:`\standardInputSpace` is the Cartesian product of vectors which 
+have components in the group :math:`\vect{u}` and other components, 
+i.e. assume that:
+
+.. math::
+
+    \standardInputSpace = \standardInputSpace_{\vect{u}} \times \standardInputSpace_{\overline{\vect{u}}}
+
+where :math:`\standardInputSpace_{\vect{u}} \subseteq \Rset^{|\vect{u}|}` 
+and :math:`\standardInputSpace_{\overline{\vect{u}}} \subseteq \Rset^{|\overline{\vect{u}}|}`. 
+We consider the conditional expectation of the function :math:`h` 
+given :math:`\standardInputObservation_{\vect{u}}`:
+
+.. math::
+
+    \widehat{h}(\standardInputObservation_{\vect{u}})
+    = \mathbb{E}_{\standardInputRandomVector_{\overline{\vect{u}}}} 
+    \left[h\left(\standardInputRandomVector\right) | \standardInputRandomVector_{\vect{u}} 
+    = \standardInputObservation_{\vect{u}}\right]
+
+for any :math:`\standardInputObservation_{\vect{u}} \in \standardInputSpace_{\vect{u}}`. 
+Let :math:`\mathcal{J}_{\vect{u}} \subseteq \mathcal{J}^P` be the 
+set of multi-indices having zero components when the marginal multi-index 
+is not in :math:`\vect{u}`:
+
+.. math::
+
+    \mathcal{J}_{\vect{u}}^{\operatorname{ce}} 
+    = \left\{\vect{\alpha} \in \mathcal{J}^P \; | \; 
+    \alpha_i = 0 \textrm{ if } i \not \in \vect{u}, \; i = 1, ..., \physicalInputDimension\right\}.
+
+This set of multi-indices defines the functions that depends on the
+variables in the group :math:`\vect{u}` and *only* them.
+Therefore :
+
+.. math::
+
+    \widehat{h}(\standardInputObservation_{\vect{u}})
+    = \sum_{\vect{\alpha} \in \mathcal{J}_{\vect{u}}^{\operatorname{ce}}}
+    a_{\vect{\alpha}} \widehat{\psi}_{\vect{\alpha}}(\standardInputObservation_{\vect{u}})
+
+for any :math:`\standardInputObservation_{\vect{u}} \in \standardInputSpace_{\vect{u}}` 
+where :math:`\widehat{\psi}_{\vect{\alpha}}` is the orthogonal polynomial defined by :
+
+.. math::
+
+    \widehat{\psi}_{\vect{\alpha}}(\standardInputObservation_{\vect{u}})
+    = 
+    \begin{cases}
+    \prod_{\substack{i = 1 \\ i \in \vect{u}}}^\physicalInputDimension \pi_{\alpha_i}^{(i)} (z_i) & \textrm{if } \alpha_i = 0 \textrm{ for any } i \not \in \vect{u},. \\
+    1 & \textrm{if } \alpha_i = 0 \textrm{ for } i = 1, ..., \physicalInputDimension, \\
+    0 & \textrm{otherwise}.
+    \end{cases}
+
+Parameters
+----------
+conditioningIndices: sequence of int in [0, inputDimension - 1]
+    The indices :math:`\vect{u}` of the input random vector to condition.
+
+Returns
+-------
+conditionalPCE: FunctionalChaosResult
```

Comment:
class openturns.FunctionalChaosResult

Reply:
and spaces around : also

Reply:
Ok, fixed at a7a2d782211ea61409b9a0f85cb552473872a914.